Libraries/Network/NetInfo.js

```diff
@@ -124,6 +125,7 @@ var _isConnectedSubscriptions = new Map();
  * - `WIFI` - The WIFI data connection.
  * - `WIMAX` -  The WiMAX data connection.
  * - `UNKNOWN` - Unknown data connection.
```

Comment:
Not sure why this list doesn't render as a list.

There's a list here:
https://facebook.github.io/react-native/docs/progressbarandroid.html#content
(source: https://github.com/facebook/react-native/blob/master/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js)